packages/fleather/test/widgets/controller_test.dart

```diff
@@ -274,6 +274,23 @@ void main() {
       // expect(controller.lastChangeSource, ChangeSource.local);
     });
 
+    test('clear', () {
```

Comment:
Shouldn't we test with `closeDocument=false`?

Reply:
Yea, I forgot about that!